ast/index.go

```diff
@@ -64,16 +63,6 @@ func (n *Index) typeMap(variable Variable, variableName string) (Type, error) {
 	return VariableMapValueTypesAreHomogenous(variableName, vmap)
 }
 
-func reportTypes(typesFound map[Type]struct{}) string {
-	stringTypes := make([]string, len(typesFound))
-	i := 0
-	for k := range typesFound {
-		stringTypes[0] = k.String()
-		i++
-	}
-	return strings.Join(stringTypes, ", ")
-}
-
```

Comment:
Fixes:
```
ast/index.go:67:6: func reportTypes is unused (unused)
func reportTypes(typesFound map[Type]struct{}) string {
```

This function is 9 years old. It is unused and doesn't add any value.